doc/user/interop.md

```diff
@@ -200,6 +200,25 @@ Otherwise throws `UnknownIdentifierException`.
 
 In all cases where a call is made no block is passed.
 
+
+### `REMOVE`
+
+The name must be an `int` (small Ruby `Fixnum`), or a Ruby `String` or `Symbol`,
+or a Java `String`.
+
+If the receiver is a Ruby `Array` and the name is an integer, delete the element
+at the index indicated by the name value. If the index is out of bounds then, in
+keeping with Ruby semantics, no-op. If the name value is not an integer, then
+`UnknownIdentifierException` is thrown.
+
+If the receiver is a Ruby `Hash`, delete the key indicated by the name value.
+If no such key exists then, in keeping with Ruby semantics, no-op.
+
+Any exception thrown during a `REMOVE` operation will be converted to `UnknownIdentifierException`.
+****
```

Comment:
What's the horizontal line for? We don't use these anywhere else in the docs, and with GitHub styling they're very bold - bolder than the lines under headers. I think it looks out of place.

Reply:
I don't know where that came from. I'll delete.